README.md

```diff
@@ -23,25 +25,95 @@ Use `pip install` to install the current project (`"."`) in editable mode (`-e`)
+## Contribution Guide 
 
-### Adding tests
-You should also add tests. These can be [doctests](https://docs.python.org/3/library/doctest.html) or as test cases in `tests/test_your_contribution_name.py`. 
+### Step 1: Choose Feature Category
 
-### Adding documentation
-***Whenever possible stick to the templates provided to ensure coherent documentation***
-- You may document your contribution in `docs/index.md`. You can also add new pages in the `docs` folder
-  - It is recommended to include at least a `index.md` file with an overview and a `quick-start.md` file, where the setup is explained.
-  - It is also very useful for users to include at least one example in form of a jupiter notebook. If you want to include a basic example, please use the basic-usage.ipynb file. You can add more examples (if possible, please replace the names of the additional-example to something more descriptive)
-- Update the `mkdocs.yml` file to reflect structure of the documentation. You can add additional pages and examples here.
-- Update the README.md file 
+First you have to choose **which type of feature** you would like to add to AutoRA. There are four categories of contributions:<br>
 
-## Add new dependencies 
+(1)  **Theorist**: An sklearn regressors that returns an interpretable model relating experiment conditions $X$ to 
+observables $Y$.<br>
+*Example: The [Bayesian Machine Scientist](https://github.com/AutoResearch/autora-theorist-bms) (Guimerà et al., 2020, 
+in Science Advances) returns an equation governing the relationship between $X$ and $Y$.* <br>
+
+(2)  **Experimentalist**: A method that identifies novel experiment conditions $X'$ that yield scientific merit. 
+Experimentalists may either be implemented as a *pooler* (generating a pool of novel experiment conditions) or as a 
+sampler (selecting from an existing pool of experiment conditions $X$).<br>
+*Example: The [Novelty Sampler](https://github.com/AutoResearch/autora-novelty-sampler) selects novel experiment 
+conditions $X'$ with respect to a pairwise distance metric applied to existing experiment conditions $X$.*
+<br>
+
+(3) **Experiment Runners**: A method that orchestrates the collecting of observations for a given set of 
+experiment conditions.<br>
+*Example: The [Firebase-Prolific Runner](https://github.com/AutoResearch/autora-experiment-runner-firebase-prolific) 
+enables the collection of behavioral data from human participants via web-based experiments hosted on 
+[Firebase](https://firebase.google.com/), using a pool of participants registered through
+[Prolific](https://www.prolific.co/).* <br>
+
+(3) **Synthetic Data**: A ground-truth model that implements a hypothesized relationship between experimental conditions
+$X$ and observations $Y$. Synthetic models may act as objects of study for which the underlying mechanisms are known, 
```

Comment:
are 'observations' and 'observables' interchangeable for us (see Theorist above)? think so in this context, just wanted to be sure